config/crds/hive_v1alpha1_clusterdeployment.yaml

```diff
@@ -46,6 +46,32 @@ spec:
               description: BaseDomain is the base domain to which the cluster should
                 belong.
               type: string
+            certificateBundles:
+              description: CertificateBundles is a list of certificate bundles associated
+                with this cluster
+              items:
+                properties:
+                  generate:
+                    description: Generate indicates whether this bundle should be
+                      generated by the system.
+                    type: boolean
+                  name:
+                    description: Name is an identifier that must be unique within
```

Comment:
should the text really read "the bundle *must* be referenced by an ingress or by the control plane serving certs" or should it read "the bundle *can* be referenced by an ingress..."?

Reply:
We’d said that an unreferenced cert bundle would result in a validation error but I’m open to changing that.

Reply:
if that's what we said, i believe you :) it's easier to loosen the restriction going forward if we need to.

Reply:
I remember that as well. If we start by requiring it, that allows us to relax it in the future if there's a useful use case.